tests/integration/inhibit-if-kmods-is-not-supported/main.fmf

```diff
@@ -0,0 +1,9 @@
+summary: kmods tests
+# TODO Add prepare with running ansible playbooks
```

Comment:
I read thorugh https://tmt.readthedocs.io/en/stable/spec/steps.html#ansible and unable to find how to specify the path to the playbooks

Reply:
I believe putting there directly the relative path from the project root should be fine:

```
prepare:
    how: ansible
    playbook:
        - some-path/common.yml
```

Reply:
Hm, this should be better documented: https://github.com/psss/tmt/pull/599

Reply:
ok, clear now. Thx